### PR TITLE
Fix version and add janjagusch as maintainer

### PR DESCRIPTION
**Pull request by janjagusch**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by janjagusch** [comment]
@conda-forge-admin, please rerender

**Review by xhochy — APPROVED** [review]

**Comment by janjagusch** [comment]
I'm a bit confused on how this happened, but the version string seems wrong. It's set to `12.0.0` but it should be `1.12.0`.

In CI we get the following error: `WARNING:conda_build.source:Error: HTTP 404 NOT FOUND for url <[https://github.com/onnx/onnx/archive/v12.0.0.tar.gz>](https://github.com/onnx/onnx/archive/v12.0.0.tar.gz%3E)`

Last release is 1.12.0, as shown here: https://github.com/onnx/onnx/releases/tag/v1.12.0

**Comment by janjagusch** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/onnx/onnx/archive/v{{ version }}.tar.gz
-  sha256: 362f7d807e899e4ab0115f2685acded6cdb0517e1f86ed7593957185b5ddb619
+  sha256: 052ad3d5dad358a33606e0fc89483f8150bb0655c99b12a43aa58b5b7f0cc507
 
 build:
   number: 0
```
Comment (xhochy) [inline recipe/meta.yaml:13]: Please bump the build no before merging. There are some slight compiler changes in this PR.

Comment (janjagusch) [reply recipe/meta.yaml:13]: Are you sure? We currently don't have a build for v1.12.0 (previously, it was v1.11.0). So shouldn't this be the same as a version bump and hence we should reset the build number to 0?

Comment (xhochy) [reply recipe/meta.yaml:13]: Ah, then it's OK.

**Pull request merged**